tests/test_cascading_delete.py

```diff
@@ -27,8 +27,19 @@ def test_delete_tree():
 
     @staticmethod
     def test_stepwise_delete():
-        assert_false(dj.config['safemode'], 'safemode must be off for testing') #TODO: just turn it off instead of warning
-        assert_true(L() and A() and B() and B.C(), 'schema population failed as a precondition to test')
+        assert_false(dj.config['safemode'], 'safemode must be off for testing')
```

Comment:
That's a lot of changes that are not relevant to this PR. All our tests are designed for nosetests. However, I will modify this specific test to switch to `assert` statements.

Reply:
@dimitri-yatsenko All I meant is for at least the newly introduced tests. We should just be mindful of this in the future is all. `nosetests` will break one day since it is [no longer maintained](https://nose.readthedocs.io/en/latest/#note-to-users) ...